docs/user_guide/bank/getting_started_customization.md

```diff
@@ -182,6 +182,16 @@ Partners can't submit Requests until they are approved by the bank.
 The full partner approval process requires the partner to fill in their profile and submit it for approval.  Some banks handle that for their partners,  gather the information through other means (such as a phone conversation). 
 Checking this will change the process so that the partners are automatically approved when they are invited. Note that any invited partners that are not yet approved will still need to be approved by the bank.
 
+## Configuring exports
+
+### Add In-Kind Value for each item
```

Comment:
Make this the whole field name -- i.e.  Include in-kind value in donation and distribution exports?

This matches the practice for the field-level sections in the rest of the file.